src/sql/workbench/services/notebook/common/notebookContext.ts

```diff
@@ -11,3 +11,8 @@ import { RawContextKey } from 'vs/platform/contextkey/common/contextkey';
 export const notebookEditorVisibleId = 'notebookEditorVisible';
 
 export const NotebookEditorVisibleContext = new RawContextKey<boolean>(notebookEditorVisibleId, false);
+
+export const NOTEBOOK_COMMAND_SEARCH = 'notebook.action.search';
```

Comment:
if its a command can we use the string 'command' instead of 'action' for clarity?